src/server/game/Entities/GameObject/GameObject.cpp

```diff
@@ -1496,8 +1513,8 @@ void GameObject::ActivateObject(GameObjectActions action, WorldObject* spellCast
 
             break;
         }
-        case GameObjectActions::None:
-            TC_LOG_FATAL("spell", "Spell %d has action type NONE in effect %d", spellId, effectIndex);
+        case GameObjectActions::SetTapList:
+            // No use cases, implementation unknown
```

Comment:
This is supposed to restrict who can loot a chest, for example with boss loot

Reply:
There also Set Tapped to Challenge Players

Lots of things in GameObject::ActivateObject depends on how [step 4](https://github.com/TrinityCore/TrinityCore/issues/21078#issuecomment-420115205) should be implemented, I guess